docs/README.md

```diff
@@ -0,0 +1,20 @@
+
+## Creating or editing Mermaid diagrams
+
+Mermaid is a JS based diagramming tool that renders markdown style syntax to create/modify diagrams. Mermaid has [native support in Github](https://github.com/github/roadmap/issues/372)
+
+## Existing Diagrams
```

Comment:
I'd move it to top level, so people see diagrams first and then how to update them

Reply:
done